catroid/src/main/res/values/strings.xml

```diff
@@ -901,6 +901,8 @@
     <string name="show_password">Show Password</string>
     <string name="register_password_mismatch">Passwords do not match</string>
     <string name="sign_in_error">Error during Sign-In</string>
+    <string name="error_connecting_to">Error: connecting failed to %s : %d%%</string>
```

Comment:
Either use
* `Failed to connect to %s : %d`
* `Connection to %s : %d failed`
* `Couldn't connect to %s : %d`
* or something similar

I could swear that the [material design writing guidelines](https://material.io/design/usability/accessibility.html#writing) have been much more precise on such strings. From a consistency point of view it doesn't matter which one you choose, we use all styles :smile:

IMO it should be `Couldn't connect to Raspberry Pi` (optionally with host and port). What do you think?

(Adding/changing string resources is expensive. It takes a long time until strings are translated to all languages. So we should always choose strings carefully!)

Reply:
btw, is there a better place for that string? All other strings in this block deal with login user accounts (login, register) only.